app/_layouts/extension.html

```diff
@@ -178,21 +239,37 @@
   {% if config_required_fields_yaml == 'config: ' %}config:
     <optional_parameter>: <value>{% else %}{{ config_required_fields_yaml }}{% endif %}
 ```
-{% endnavtab %}
-{% endunless %}
-{% endnavtabs %}
-{% endif %}
 
-`<service>` is the `id` or `name` of the Service that this plugin
+`<service>` is the `id` or `name` of the service that this plugin
   configuration will target.
 
-{% if page.params.api_id %}
-  > **Note:** The legacy API entity has been **deprecated** in favor of Services
-  since <a href="https://github.com/Kong/kong/blob/master/CHANGELOG.md#0130---20180322">
-  CE 0.13.0</a> and
-  <a href="https://docs.konghq.com/enterprise/changelog/#0-32">EE 0.32</a>.
-{% endif %}
+{% endnavtab %}
+{% endunless %}
+{% unless page.params.konnect_examples == false %}
+{% navtab Konnect %}
+Configure this plugin on a service:
+
+1. In Konnect, select the service on the ServiceHub page.
```

Comment:
```suggestion
1. In {{site.konnect_saas}}, select the service on the ServiceHub page.
```
Applies throughout, just not in tab titles - that won't get parsed.

Reply:
ah, yeah will fix